docker-jans-configurator/scripts/bootstrap.py

```
@@ -100,7 +101,8 @@ def generate_pkcs12(suffix, passwd, hostname):
         f"-passout pass:{passwd}",
     ])
     _, err, retcode = exec_cmd(cmd)
-    assert retcode == 0, f"Failed to generate PKCS12 file; reason={err}"
+    if retcode != 0:
+        raise CalledProcessError(f"Failed to generate PKCS12 file; reason={err.decode()}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)